src/shared/api/CivicAPI.ts

```diff
@@ -0,0 +1,265 @@
+import $ from 'jquery';
```

Comment:
Actually the content of this file looks like CivicAPI + CivicUtils to me. I mean it both contains the data retrieval functions (API) and data processing functions (Utils). Might be better to separate them.